### DISCUSSION TITLE
Blood/plasma concentration ratio has no impact on PK in PK-Sim standard model

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#897] [Questions & Problems]
Dear all,

When using in a model the PK-Sim standard distribution model, I tried to modify the parameter "Blood/plasma concentration ratio", and simulating both the output "peripheral venous blood/whole blood" and "PVB/plasma". By changing the "Blood/plasma concentration ratio" there is no difference in the PK profiles generated, which I find very odd. Do you have any explanation? I could replicate this behavior also in the example model file for paracetamol provided with PK-Sim, so this is not limited to my model. (To replicate, I just changed the distribution model from R&R to PK-Sim standard in the example). Just as comparison, with the R&R distribution model the PK in plasma and blood was modified by changing the blood/plasma parameters for this model, so what I observed seems to be specific for the PK-Sim distribution model.

Does anyone has any explanation? And even more important, is this behavior correct?

Thanks in advance for your clarifications!

Best,

Donato

**Comment by Yuri05** [comment]
Well, the explanation is: not "Blood/plasma concentration ratio" (**B2P**) itself is used in the "PK-Sim standard" distribution model, but its formula.

This is how it looks like in PK-Sim standard model:

```
Parameter: S1|C1|Blood/Plasma concentration ratio
	Formula: (f_water_rbc + f_lipids_rbc * 10^LogP + f_proteins_rbc * KProt) * fu * HCT - HCT + 1
		fu: S1|C1|Fraction unbound (plasma)(0.11)
		LogP: S1|C1|Lipophilicity(3 [Log Units])
		KProt: S1|C1|Partition coefficient (water/protein)(22.2331460674157)
		HCT: S1|Organism|Hematocrit(0.47)
		f_lipids_rbc: S1|Organism|Vf (lipid, blood cells)(0.005)
		f_proteins_rbc: S1|Organism|Vf (protein,blood cells)(0.325)
		f_water_rbc: S1|Organism|Vf (water,blood cells)(0.625)

Parameter: S1|C1|Partition coefficient (blood cells/plasma)
	Formula: (f_water_rbc + f_lipids_rbc * 10^LogMA + f_proteins_rbc * KProt)*fu
		fu: S1|C1|Fraction unbound (plasma)(0.11)
		LogMA: S1|C1|Lipophilicity(3 [Log Units])
		KProt: S1|C1|Partition coefficient (water/protein)(22.2331460674157)
		f_lipids_rbc: S1|Organism|Vf (lipid, blood cells)(0.005)
		f_proteins_rbc: S1|Organism|Vf (protein,blood cells)(0.325)
		f_water_rbc: S1|Organism|Vf (water,blood cells)(0.625)
```

As you can easily see, in fact it is `Partition coefficient (blood cells/plasma) = (HCT - 1 + B2P) / HCT`
But due to the fact that B2P is not used **directly**, changing it does not have any effect.

Comparing it with e.g. Rogers and Rowland model:
```
Parameter: S2|C1|Blood/Plasma concentration ratio
	Formula: (f_water_rbc + f_lipids_rbc * 10^LogP + f_proteins_rbc * KProt) * fu * HCT - HCT + 1
		fu: S2|C1|Fraction unbound (plasma)(0.11)
		LogP: S2|C1|Lipophilicity(3 [Log Units])
		KProt: S2|C1|Partition coefficient (water/protein)(22.2331460674157)
		HCT: S2|Organism|Hematocrit(0.47)
		f_lipids_rbc: S2|Organism|Vf (lipid, blood cells)(0.005)
		f_proteins_rbc: S2|Organism|Vf (protein,blood cells)(0.325)
		f_water_rbc: S2|Organism|Vf (water,blood cells)(0.625)

Parameter: S2|C1|Partition coefficient (blood cells/plasma unbound)
	Formula: (HCT - 1 + B2P) / fu / HCT
		B2P: S2|C1|Blood/Plasma concentration ratio(1.19438493679775)
		fu: S2|C1|Fraction unbound (plasma)(0.11)
		HCT: S2|Organism|Hematocrit(0.47)

Parameter: S2|C1|Partition coefficient (blood cells/plasma)
	Formula: K_pu_bc*fu
		fu: S2|C1|Fraction unbound (plasma)(0.11)
		K_pu_bc: S2|C1|Partition coefficient (blood cells/plasma unbound)(12.8507724719101)
```
As you can easily see from the last 2 equations
`Partition coefficient (blood cells/plasma) = (HCT - 1 + B2P) / HCT`

Which is exactly the same as in the PK-Sim standard model. Just with the difference that B2P i used **directly**.

I guess this is due to historical reasons. "PK-Sim standard" was the first distribution model used in PK-Sim, R&R was added later. 
"PK-Sim standrad" did not rely on B2P, which was introduced with R&R. "PK-Sim standard" was left AS IS.
As far as I remember, in PK-Sim 4 B2P was hidden in case "PK-Sim standard" distribution was selected - to avoid thid kind of confusion.

Don't remember why "PK-Sim standard" was kept AS IS - should be doable to rewrite the equations in the same way as in R&R.

**Comment by teutonicod** [comment]
Hi @Yuri05 ,

Thanks a lot for the explanation. I would say that if possible, it would be better to use directly the value provided, instead of the formula, in the PK-Sim dist model. Since it would avoid misunderstanding on one side, and most importantly, if someone has measured values for B/P partition (this is my case :-)) it would be possible to integrate this information in the distribution model, as it is already possible in the other distribution model.

**Comment by Yuri05** [comment]
Created an issue for this: https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1455

**Comment by teutonicod** [comment]
Thanks a lot @Yuri05 !
Is there a way to change the blood to plasma ratio in a simulation so that is would be used instead of the formula? in PK-Sim (ideally) or in MoBi?

**Comment by Yuri05** [comment]
>Is there a way to change the blood to plasma ratio in a simulation so that is would be used instead of the formula?

You mean right now? 
 * If you need to do it in PK-Sim, you could just calculate of `(HCT - 1 + B2P) / HCT` and set it as the new value for `Partition coefficient (blood cells/plasma)` (of course as soon as HCT or B2P changes it must be redone...). Better solution not possible.
* In MoBi: In the Spatial Structure Building block, change the formula of `Partition coefficient (blood cells/plasma)`
![grafik](https://user-images.githubusercontent.com/25061876/77556268-91f4d180-6eb8-11ea-9569-aaa8f9b06920.png)
1. Change the formula type to "Explicit ormula"
2. Create new formula and set it to `(HCT - 1 + B2P) / HCT` (s. above)

**Reply by moPellowe** [reply]
I have a follow-up question to this. If I wanted to apply the formula (as above) to a population, then I would add/update the "Partition coefficient (blood cells/plasma)" column within the population csv, right, since each individual would have their own HCT value?

**Reply by Yuri05** [reply]
In this case you will need to import/send the modified simulation to PK-Sim and combine it there with a population.
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/importing-exporting-project-data-models#importing-individual-and-population-simulation

**Reply by prvmalik** [reply]
In the population simulation I think you also have to address the permeability into and out of red blood cells if I recall, and not just the partition coefficient. Have a look at what parameters are different in the individual sim vs. the population sim when you manually tune B2P.

**Reply by moPellowe** [reply]
So this requires changing the formula in MoBi, but is there a way to avoid MoBi altogether and just work with the population csv? I could theoretically over-write any parameter by adding a column to the exported population and re-importing, right?

**Reply by Yuri05** [reply]
If losing the formula dependencies is OK: you can do it.

**Comment by teutonicod** [comment]
OK, thanks a lot!

**Comment by Erik-Sjogren** [comment]
Sorry for being late into this discussion.
Is there a plan for doing this more convenient? 
Lets say have an option for that if one manually input B:P ratio (at given HCT) in Compound BB, the "Partition coefficient (blood cells/plasma)" is automatically calculated and used for all simulations?

**Reply by StephanSchaller** [reply]
Hi Eric, yes, that is what Juri created the issue for above (https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1455)

**Reply by Erik-Sjogren** [reply]
Sorry, I am a bit confused on how to interpret your comment. 
Is PK-Sim#1455 where discussions on this topic should happen?
Should I re-post my comment there?
Or is PK-Sim#1455 a place holder for an action/task?

**Reply by StephanSchaller** [reply]
> Or is PK-Sim#1455 a place holder for an action/task?

yes, this is the feature to also add this to the "PK-Sim standard" model. As Juri showed above, this is already the case if you use RR.

**Reply by Erik-Sjogren** [reply]
Sorry for continuing on the topic.
I cannot find (in this thread) that the functionality/feature I suggest is, or will be, implemented (regardless of distribution model).
Are you referring to including "fu" in the B:P equation?
Anyways, as it is now (PK-Sim v9), and correct me if I'm wrong, there is no possibility to add B:P info in the compound BB that will override the predicted "Partition coefficient (blood cells/plasma)", regardless of which distribution model you select. This needs to be changed in the simulation
Also, the parameter "Blood/Plasma concentration ratio" has only implications for the simulations using RR.

**Reply by StephanSchaller** [reply]
Well, the feature issue created by Juri (#1455) is to make the behavior of "PK-Sim standard" equal to RR.

This is what happens in PK-Sim for PC method RR:

![image](https://user-images.githubusercontent.com/25477643/118807736-f8049b00-b8a8-11eb-8542-735d0aec8c12.png)

So your B:P info goes into "Blood/Plasma concentration ratio" and affects "Partition coefficient (blood cells/plasma)". But yes, it is not a property you can set in the compound building block... yet (and "having "Blood/Plasma concentration ratio" as a (input) property you can set in the compound building block" is also not yet formulated as a feature request, we could make this an optional input which then overwrites the LogP dependency of B/P partitioning?...).

**Reply by Erik-Sjogren** [reply]
Hi,
You are faster than lightning.. :).
I was updating my post related to the RR... but you beat me to it.

Ok, agreeing on that. The aspect of potentially including an override in compound BB still remains.

BR
E

**Reply by StephanSchaller** [reply]
Haha, I wish...pure luck.

Okay, agreed.

**Comment by PavelBal** [comment]
Let's reopen, @Erik-Sjogren has a valid point.
